### ISSUE TITLE
loading message on multiple attributs edition

### ISSUE THREAD

**Issue by truckydev** [MISP/MISP#1376]
Hi team,

When editing multiple attributes on an event, the loading message is displayed without success / failure notification .
### Work environment

| Questions | Answers |
| --- | --- |
| Type of issue | Bug |
| MISP version / git hash | 2.4.48 |
### Expected behavior

You can update Distribution, toIDS, and comment for selected elements.
### Actual behavior

"loading" message appears
### Steps to reproduce the behavior

On an event , select one or more items . And try to change them.
### Logs, screenshots, configuration dump, ...

![animated](https://cloud.githubusercontent.com/assets/17462115/17018102/7d322edc-4f36-11e6-9189-5fa90984a497.gif)
### Feature idea :

It would be interesting to concatenate comment  attribute with the comment field in the dialog box . To avoid losing information.

**Comment by iglocska** [comment]
Indeed, there is no message with the outcome so that needs to be fixed. But other than the message the updates work, right?

**Comment by truckydev** [comment]
Nop :(

**Comment by iglocska** [comment]
Hmph. It saves it correctly for me, could you check your error.log if anything shows up?

**Comment by truckydev** [comment]
Nothing for apache, mysql log :(,

My apache is in debug mod.

**Comment by iglocska** [comment]
Sorry, I meant in the misp error logs

/var/www/MISP/app/tmp/logs/error.log

**Comment by truckydev** [comment]
Nothing either in this log file. I try on other instance starting from a clean install and I have the same problem.

**Comment by elhoim** [comment]
No special plugin or browser feature to disable JS?

**Comment by truckydev** [comment]
Nop :(
I try with other browser (firefox ESR, chromium), OS on other instance too. And I have the same
